### PR TITLE
legacy scheduler

### PR DESCRIPTION
**Pull request by StacyYang**

### CODE REVIEW

File: autotorch/scheduler/resource/manager.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import logging
-import multiprocessing as mp
+import torch.multiprocessing as mp
 from .resource import *
-from ...utils import Queue
+#from ...utils import Queue
```
Comment (StacyYang) [inline autotorch/scheduler/resource/manager.py:4]: why not using utils.Queue?
